platforms-serverless-vercel/vercel-with-redwood/api/upgrade.sh

```diff
@@ -2,4 +2,4 @@
 
 set -eu
 
-yarn policies set-version 1.18.0
+yarn policies set-version 1.22.19
```

Comment:
Remove this warning and it's the version used by most people (1.22)
```
prisma:warn Your yarn has version 1.18.0, which is outdated. Please update it to 1.19.2 or newer in order to use Prisma.
```

Reply:
What is this used/needed for? Wort adding a comment?

Reply:
Maybe we don't even need this anymore, I'll try removing them in a PR

Reply:
I honestly do not know what it was for.

Reply:
We probably had an issue with some version of yarn at some point and we had this use this specific version workaround

Reply:
Did a revert, turned out to break in check-for-update, added a comment
https://github.com/prisma/ecosystem-tests/commit/88e5bca3b7cd63dd7cd5e623ac3a3953274328e2